doc/source/guidelines/PyMAPDL_WSL_Guide.rst

```diff
@@ -0,0 +1,423 @@
+  .. _ref_guide_wsl:
+
+
+ANSYS on Windows Subsystem for Linux and Docker
+###############################################
+
+This guide shows you how to use Ansys products, more specifically PyMAPDL, in the Windows Subsystem for Linux (WSL).
+WSL is a compatibility layer for running Linux binary executables natively on Windows 10, Windows 11, and Windows Server 2019 
+(`Wikipedia-WSL <https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux>`_).
+
+This guide walks you through the installation of WSL in Windows, and shows how to use it together with ANSYS, PyMAPDL and docker.
+
+For more information about WSL please visit `What is the Windows Subsystem for Linux? <https://docs.microsoft.com/en-us/windows/wsl/about>`_. 
+
+.. warning:: **Disclaimer:** This guide is still in Alpha state. Please proceed with caution. 
+
+
+.. warning:: This guide hasn't been tested with a VPN connection.
```

Comment:
This guide hasn't been tested with a VPN connection or WSL hasn't been tested with a VPN connection?

Reply:
This guide. WSL should work, and in theory, Ansys should too, but there are some caveats that makes me think it might fail.

Reply:
I edited this:

```
.. warning:: This guide hasn't been fully tested with a VPN connection. If you experience any problems to connect WSL to internet, try to disconnect from the VPN. 
```

Reply:
This really depends on the VPN used.. WSL itself can also have issues (e.g., no internet connection) when on VPN.